src/index.ts

```diff
@@ -660,9 +676,11 @@ export const formats: IFormat[] = [
   bitcoinBase58Chain('DIVI', 301, [[0x1e]], [[0xd]]),
   getConfig('DOT', 354, dotAddrEncoder, ksmAddrDecoder),
   getConfig('KSM', 434, ksmAddrEncoder, ksmAddrDecoder),
+  getConfig('SOL', 501, bs58Encode, bs58Decode),
```

Comment:
I tried to resolve conflict and put duplicate `SOL`. Can you get rid of line 679, keeping line 681?

Reply:
Umm, now the test is failing. Can you figure out why it is failing?

Reply:
Resolved.